scripts/verify.sh

```diff
@@ -38,15 +38,15 @@ golangci-lint run --disable-all \
     --enable=nakedret \
     --enable=interfacer \
     --enable=misspell \
-    --enable=dupl
+    --enable=dupl \
+    --enable=goimports \
+
 
 ##todo(camilamacedo86): The following checks requires fixes in the code
 # --enable=golint
 # --enable=gocyclo
-# --enable=goimports
 # --enable=lll
 # --enable=goconst
 # --enable=gosec
 # --enable=maligned
 
-GO111MODULES=on go list -mod=readonly ./...
```

Comment:
Does `golangci-lint run --enable=goimports` supersede this check?

Reply:
You check to fast before my fix pr :-) 
Tks for the help .